Update data-retention: New FAQ (#3386)
mattermost/docs: source/administration/data-retention.rst

Frequently Asked Questions (FAQs)
---------------------------------

What happens when the data retention period is changed?
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> Data retention runs once a day at the time specified in the ``config.json`` file. Changing the retention period does not automatically schedule any additional run of the data retention job. It just updates the configured period, which is honored when the next scheduled data retention job is run.

Reviewer: ```suggestion
Data retention runs once a day at the time specified in the ``config.json`` file. Changing the retention period does not automatically schedule any additional run of the data retention job - it only updates how long data is kept in Mattermost.
```

Reviewer: Propose simplifying the language
